src/js/actions/layers.js

```diff
@@ -732,6 +732,28 @@ define(function (require, exports) {
     resetLayerVisibility.modal = true;
 
     /**
+     * Initialize the uninitialized subset of the given layers by loading their lazy properties.
+     *
+     * @param {Document} document
+     * @param {Layer|Immutable.Iterable.<Layer>} layerSpec
+     * @return {Promise}
+     */
+    var initializeLayers = function (document, layerSpec) {
```

Comment:
I just moved this up in the file so that `setGroupExpansion` could transfer to it. No changes here.